Index.html

```diff
@@ -0,0 +1,177 @@
+    <div><a href="Index.html" class="logo"><img src="img/mBlack.png" alt="M logo" class="desk-logo"></a></div>
+    <ul id="desk-ul" class="desk-ul">
+        <li class="desk-li"><a href="about.html">About</a></li>
+        <li class="desk-li"><a href="#">Course</a></li>
+        <li class="desk-li"><a href="#">FAQs</a></li>
+        <li class="desk-li"><a href="#">Sign in</a></li>
+        <li class="desk-li"><a href="#">Log in</a></li>
+    </ul>
+</section>
+  <!-- body headline part -->
+  <section class="headline headline-main">
+    <h1>
+      "Hello!! world"
+    </h1>
+    <p class="headline__p">
+      Welcome<br>to my<br>DemoCapstone.
+    </p>
+
+    <!-- box inside headline -->
+    <div class="headline__box1">
+      Lorem, ipsum dolor sit amet consectsfetur niadipisicing aselit. Sit asint, odit, placeat magni mollitia iusto hic quibusdam facere kosfni
+    </div>
+    <div class="headline__box2">
+      <h2 class="headline__box2__h2 ">
+        2023.10.05(THU) ~ 16(FRI)
+      </h2>
+      <p class="headline__box2__p">
+        @National museum Bangladesh, Art center and more.
+      </p>
+    </div>
+  </section>
+</div>
+
+<!-- Main Program part-->
+  <section class="main-program">
+    <h2 class="main-program__h2">
+      Main Program
+      <span class="red-bar"></span>
+    </h2>
+    <!-- boxes of grid -->
+    <div class="main-program__grid-container">
+    <div class="main-program__box box1">
+      <img src="img/Icon-1.svg" alt="" class="main-program__box__logo">
+      <h3 class="main-program__box__h3">
+        Lecture
+      </h3>
+      <p class="main-program__box__p">
+        Lorem, ipsum dolor sit amet consectetur adipisicing elit. Minus debitis iure reiciendis ipsa, veniam laudantium.
+      </p>
+    </div>
+
+    <!-- boxes of grid -->
+    <div class="main-program__box box2">
+      <img src="img/Icon-2.svg" alt="" class="main-program__box__logo">
+      <h3 class="main-program__box__h3">
+        Ignite
+      </h3>
+      <p class="main-program__box__p">
+        Lorem, ipsum dolor sit amet consectetur adipisicing elit. Minus debitis iure reiciendis ipsa, veniam laudantium.
+      </p>
+    </div>
+
+    <!-- boxes of grid -->
+    <div class="main-program__box box3">
+      <img src="img/Icon-3.svg" alt="" class="main-program__box__logo">
+      <h3 class="main-program__box__h3">
+        Forum
+      </h3>
+      <p class="main-program__box__p">
+        Lorem, ipsum dolor sit amet consectetur adipisicing elit. Minus debitis iure reiciendis ipsa, veniam laudantium.
+      </p>
+    </div>
+
+    <!-- boxes of grid -->
+    <div class="main-program__box box4">
+      <img src="img/Icon-1.svg" alt="" class="main-program__box__logo">
+      <h3 class="main-program__box__h3">
+        Workshop
+      </h3>
+      <p class="main-program__box__p">
+        Lorem, ipsum dolor sit amet consectetur adipisicing elit. Minus debitis iure reiciendis ipsa, veniam laudantium.
+      </p>
+    </div>
+    </div>
```

Comment:
- [ ] Your middle section is great and very similar to the template, but there are only 4 cards. Kindly add one more card to match the 5 cards of the template. Please make sure the alignment, sizing, and responsiveness remain the same.
 - Your design:
 - 
<img width="1112" alt="image" src="https://user-images.githubusercontent.com/61919617/217387754-fdb01083-d5b6-4f3f-977c-700aabb4b885.png">

 - Template:
 - 
<img width="1033" alt="image" src="https://user-images.githubusercontent.com/61919617/217387815-4c1ddc29-30f2-4c9f-aa27-b9bfe1f6aab6.png">